libafl_bolts/src/serdeany.rs

```diff
@@ -98,7 +140,7 @@ pub mod serdeany_registry {
         where
             V: serde::de::SeqAccess<'de>,
         {
-            let id: u128 = visitor.next_element()?.unwrap();
+            let id: TypeRepr = visitor.next_element()?.unwrap();
```

Comment:
@addisoncrump I'm not 100% sure if this type annotation is correct (or should be borrowed or something?)